zcutil/build.sh

```diff
@@ -40,6 +40,14 @@ if [ -z "${CONFIGURE_FLAGS-}" ]; then
     CONFIGURE_FLAGS=""
 fi
 
+case "$@" in
+    *"V=1"*)
+        ;;
+    *)
+        CONFIGURE_FLAGS="--quiet $CONFIGURE_FLAGS"
+        ;;
+esac
```

Comment:
Ah, thanks. I had tried something else first, but it didn’t work because !Bash. Then I just copied what was done a few lines lower for setting `DEBUG`, because I couldn’t remember what stuff was Bash-specific. Should I also switch the `DEBUG` one to use `for`? It currently looks like
```shell
case "$CONFIGURE_FLAGS" in
(*"--enable-debug"*)
    DEBUG=1
;;
(*)
    DEBUG=
;;esac
```

I generally prefer additive semantics over replacement, since I think it’s more obvious to users. But `./configure` dosen’t offer any inverse to `--quiet` like `--verbose` or `--no-quiet` (I had assumed it did), so I think in this case you’re right. /me starts drafting a blog post titled “command-line options should form a group”.